modules/widgets/blog-stats.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Blog Stats Widget.
+ *
+ * @since 4.5.0
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Blog Stats Widget.
+ *
+ * Displays all time stats for that site.
+ *
+ * @since 4.5.0
+ */
+class Jetpack_Blog_Stats_Widget extends WP_Widget {
+
+	/**
+	 * Constructor
+	 */
+	function __construct() {
+		$widget_ops = array(
+			'classname' => 'blog-stats',
+			'description' => __( 'Show a hit counter for your blog.', 'jetpack' ),
```

Comment:
In case translations somehow introduce a security flawed string?  Sure, doesn't hurt to escape it.